### PR TITLE
Add actual list of redirectors currently active

### PR DESCRIPTION
**Pull request by dims**
We don't have a list of these url(s) anywhere on a single page which
makes it harder to discover

### CODE REVIEW

File: k8s.io/README.md [hunk 1]
```diff
@@ -3,6 +3,20 @@ Overview
 This contains the Nginx configuration for k8s.io and the associated subdomain
 redirectors.
 
+Redirections
```
Comment (ixdy) [inline k8s.io/README.md]: maybe add a comment in the ngnix configmap to keep this list up-to-date?

Comment (fejta) [reply k8s.io/README.md]: Also link to the configmap so someone can find the source of truth?

Comment (thockin) [reply k8s.io/README.md]: We should also document the vanity URLs we support (there are a lot of them, eg. pr.k8s.io)

Comment (dims) [reply k8s.io/README.md]: Ack @thockin i'll figure that out

**Comment by jdumars** [comment]
Thanks @dims !

**Pull request merged**